src/utilities/util_dump.c

```diff
@@ -146,12 +165,34 @@ util_dump(WT_SESSION *session, int argc, char *argv[])
         ret = util_err(session, errno, NULL);
 
     __wt_scr_free(session_impl, &tmp);
+    __wt_scr_free(session_impl, &tmp2);
+    __wt_scr_free(session_impl, &timestamp_buf);
     free(uri);
     free(simpleuri);
 
     return (ret);
 }
 
+/*
+ * time_pair_to_timestamp --
+ *     Convert a timestamp output format to timestamp representation.
+ */
+static int
+time_pair_to_timestamp(WT_SESSION_IMPL *session_impl, char *ts_string, WT_ITEM *buf)
+{
+    uint64_t timestamp;
+    uint32_t first, second;
+
+    if (ts_string[0] == '(') {
+        sscanf(ts_string, "(%u,%u)", &first, &second);
+        timestamp = ((unsigned long long)first << 32) | second;
+    } else
+        timestamp = strtoul(ts_string, NULL, 10);
+
+    WT_RET(__wt_buf_catfmt(session_impl, buf, "%lx", timestamp));
+    return 0;
+}
```

Comment:
Will users enter base-10 timetamps, or base-16? (I have no idea, just asking.)

Reply:
Hey Keith, users will be enter base-10 timestamps

Reply:
I honestly don't know -- we **display** timestamps in hex, right? I guess the question is, where did the user get the timestamp they're entering on the command line? From an MongoDB/WiredTiger log message, or some other way? If there's a leading `0x`, I think sscanf does the right thing by default, but I'm not sure I'm remembering that correctly.

Reply:
When we display timestamps in the utility tool, in both **dump and verify** we display them in (%PRIu32,%PRIu32) format

Reply:
I had in mind that we input in the same format that we display MongoDB timestamps. That way we can copy paste from a log message and use the time stamp as such.